fastqheat/__main__.py

```diff
@@ -60,194 +54,230 @@ def _make_accession_list(term: str) -> list[str]:
+    """
+    This help message is also accessible via `python3 -m fastqheat --help`.
 
+    ## Compatibility
 
-def handle_methods(term: str, method: str, out: PathType, *, core_count: int) -> list:
-    """Runs specific download function based on the given method."""
+    FastqHeat is being developed and tested under Python 3.9.x.
 
-    states = []
-    # try:
-    #     download_function = method_to_download_function[
-    #         method
-    #     ]
-    # except KeyError:
-    #     raise ValueError(f"Unknown method: {method}")
+    ## Installation
 
-    accession_list = _make_accession_list(term)
+     1. [Make sure you have installed a supported version of Python](https://www.python.org/downloads/).
 
-    if method == 'q':
-        for accession in accession_list:
-            states.append(download_function(accession, out, core_count=core_count))
-    else:
-        for accession in accession_list:
-            states.append(download_function(accession, out))
+     2. Clone this project from GitHub or download it as an archive.
 
-    return states
+     3. **Optional, but recommended:** create and activate a fresh
+     [virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments).
 
+     4. Install it directly with `pip`.
 
-class TermParser:
-    """
-    Provides a method for parsing terms from a string.
+    Full example for Linux systems:
+
+    bash:
+
+    $ git clone git@github.com:quantori/FastqHeat.git
+
+    $ python3 -m venv env
+
+    $ . env/bin/activate
 
-    A string can represent a singular term or a *.txt file with a few terms.
+    $ pip install FastqHeat/
     """
+    pass
+
+
+@click.command()
+@common_options
+@click.option(
+    '--transport',
+    default='binary',
+    show_default=True,
+    help='Transport (method) to be user to download data.',
+    type=click.Choice(['binary', 'ftp'], case_sensitive=False),
+)
+@click.option(
+    '--metadata-file',
+    default=get_metadata_file,
+    show_default=True,
+    help='Metadata filepath',
+    type=click.Path(exists=False, file_okay=True, dir_okay=False, writable=True),
+)
+@click.option(
+    '--skip-download-metadata',
+    default=False,
+    show_default=True,
+    help='Skip metadata download step',
+)
+def ena(
+    working_dir: str,
+    metadata_file: str,
+    config: configparser.ConfigParser,
+    transport: str,
+    accession: list[str],
+    attempts: int,
+    attempts_interval: int,
+    cpu_count: int,
+    skip_download: bool,
+    skip_check: bool,
+    skip_download_metadata: bool,
+) -> None:
+    if not skip_download:
+        ena_module.download(
+            accessions=accession,
+            output_directory=working_dir,
+            transport=transport,
+            binary_path=config['NCBI']['FasterQDump'],
+            attempts=attempts,
+            attempts_interval=attempts_interval,
+            cpu_count=cpu_count,
+            aspera_ssh_path=config['ENA']['asperaweb_id_dsa.openssh']
```

Comment:
I guess you meant to write `aspera_ssh_path=config['ENA']['asperaweb_id_dsa.openssh']` otherwise `KeyError: 'asperaweb_id_dsa.openssh'`